docs/kafka/manage-cluster/README.adoc

```diff
@@ -0,0 +1,250 @@
+:registry: Service Registry
+:product-version-registry: 1
+:service-url-registry: https://console.redhat.com/application-services/service-registry/
+:getting-started-url-registry: registry/getting-started-registry/README.adoc
+:quarkus-url-registry: registry/quarkus-registry/README.adoc
+:getting-started-rhoas-cli-url-registry: registry/rhoas-cli-getting-started-registry/README.adoc
+:access-mgmt-url-registry: registry/access-mgmt-registry/README.adoc
+:content-rules-registry: https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry/1/guide/9b0fdf14-f0d6-4d7f-8637-3ac9e2069817[Supported Service Registry content and rules]
+:service-binding-url-registry: registry/service-binding-registry/README.adoc
+
+//OpenShift Connectors
+:connectors: Connectors
+:product-long-connectors: OpenShift Connectors
+:product-connectors: Connectors
+:product-version-connectors: 1
+:service-url-connectors: https://console.redhat.com/application-services/connectors
+:getting-started-url-connectors: connectors/getting-started-connectors/README.adoc
+:getting-started-rhoas-cli-url-connectors: connectors/rhoas-cli-getting-started-connectors/README.adoc
+:addon-url-connectors: https://access.redhat.com/documentation/en-us/openshift_connectors/1/guide/15a79de0-8827-4bf1-b445-8e3b3eef7b01
+
+
+//OpenShift API Designer
+:product-long-api-designer: OpenShift API Designer
+:product-api-designer: API Designer
+:product-version-api-designer: 1
+:service-url-api-designer: https://console.redhat.com/application-services/api-designer/
+:getting-started-url-api-designer: api-designer/getting-started-api-designer/README.adoc
+
+//OpenShift API Management
+:product-long-api-management: OpenShift API Management
+:product-api-management: API Management
+:product-version-api-management: 1
+:service-url-api-management: https://console.redhat.com/application-services/api-management/
+
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-managing-cluster"]
+= Managing {product-long-kafka} on a ROSA cluster
+ifdef::context[:parent-context: {context}]
+:context: deploying-cluster
+
+// Purpose statement for the assembly
+[role="_abstract"]
+
+As an organization administrator, you can deploy {product-long-kafka} within your own controlled environment on your own ROSA cluster.
+
+//Additional line break to resolve mod docs generation error.
+
+[id="proc-installing-on-your-rosa-cluster_{context}"]
+== Installing {product-kafka} on your ROSA cluster
+
+You must install {product-long-kafka} on your {rosa-name} ROSA cluster before users in your organization can create Kafka resources. Use the {product-long-rhoas} `rhoas` command-line interface (CLI) to register your (ROSA) cluster through a call to the Kafka Service Fleet Manager service. This service exposes a Rest API to manage Kafka instances. Registering your ROSA cluster allows the service to install certain necessary add-ons to your cluster so users can provision their Kafka instances.
```

Comment:
```suggestion
You must install {product-long-kafka} on your {rosa-name} ROSA cluster before users in your organization can create Kafka resources. Use the {product-long-rhoas} `rhoas` command-line interface (CLI) to register your (ROSA) cluster through a call to the Kafka Service Fleet Manager service. This service exposes a REST API to manage Kafka instances. Registering your ROSA cluster allows the service to install necessary add-ons to your cluster so users can provision their Kafka instances.
```

Reply:
+1 @k-wall I think we should only mention RHOSAK service or just service in place where we mention fleet manager

Reply:
Thanks @k-wall. I will remove any mention of it from the doc.

Reply:
Change looks good.